tutorials/mnist_tutorial_tf.py

```diff
@@ -21,6 +23,9 @@
 flags.DEFINE_integer('nb_epochs', 6, 'Number of epochs to train model')
 flags.DEFINE_integer('batch_size', 128, 'Size of training batches')
 flags.DEFINE_float('learning_rate', 0.1, 'Learning rate for training')
+flags.DEFINE_boolean('iterative', True, 'Whether to use basic iterative method'
+                                        'to craft adversarial samples. if false, '
+                                        'FGSM will be used.')
```

Comment:
Good idea. Ian may have a style preference for how this messaging is done, so I will leave it to him.